src/cata_utility.h

```diff
@@ -433,15 +433,15 @@ bool string_starts_with( const std::string &s1, const std::string &s2 );
  */
 template <std::size_t N>
 // NOLINTNEXTLINE(modernize-avoid-c-arrays)
-inline bool string_starts_with( const std::string &s1, const char( &s2 )[N] )
+inline bool string_starts_with( const std::string_view s1, const char( &s2 )[N] )
```

Comment:
Is this still needed if you make the one above inline/constexpr? They look the same on godbolt https://godbolt.org/z/9K5dafK88

Reply:
Looks like you're right (I checked they were still the same even on gcc 9.2).  I'll consolidate them.  Thanks for researching that.